lib/tic_tac_toe.rb

```diff
@@ -32,28 +33,56 @@ def set_board_middle
         add_board_edge
         add_board_spaces
         add_board_edge
-        @board_middle = [@board_middle.join]
+        @board_size.times do
+            @board_middle << [@board_middle_section.join]
+        end
+    end
+
+    def reset_middle_section
+        @board_middle_section = []
     end
 
     def add_board_spaces
         @board_size.times do 
-            @board_middle.push('.')    
+            @board_middle_section.push('.')    
         end
     end
 
     def add_board_edge
-        @board_middle << BOARD_COLUMM
+        @board_middle_section << BOARD_COLUMM
     end
 
     def assemble_board
         @board << @board_top_and_bottom
-        @board_size.times do
-            @board << @board_middle
-        end
+        @board << @board_middle
         @board << @board_top_and_bottom
     end
 
     def view_board
+        system('clear')
+        reset_board
+        assemble_board
         puts @board
     end
-end
\ No newline at end of file
+
+    def reset_board
+        @board = []
+    end
+
+    def player_move
+        player_input = -1
+        while player_input < 1 || player_input > 9 do
+            puts 'Please enter a number between 1-9'
+            player_input = gets.chomp.to_i
+        end
+        make_move(player_input)
+        view_board
+    end
+
+    def make_move(square)
```

Comment:
I'm not immediately clear on what this method is doing. Is there a way to make this more readable?